WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -798,19 +825,21 @@ class AztecPostViewController: UIViewController, PostEditor {
     }
 
     func reloadPublishButton() {
-        publishButton.title = postEditorStateContext.publishButtonText
+        publishButton.setTitle(postEditorStateContext.publishButtonText, for: .normal)
         publishButton.isEnabled = postEditorStateContext.isPublishButtonEnabled
     }
 
     func resizeBlogPickerButton() {
         // Ensure the BlogPicker gets it's maximum possible size
-        blogPickerButton.sizeToFit()
-
-        // Cap the size, according to the current traits
-        var blogPickerSize = hasHorizontallyCompactView() ? Constants.blogPickerCompactSize : Constants.blogPickerRegularSize
-        blogPickerSize.width = min(blogPickerSize.width, blogPickerButton.frame.width)
-
-        blogPickerButton.frame.size = blogPickerSize
+        if #available(iOS 11, *) {
```

Comment:
Here are several iOS 11 guards... why so?  Is the issue specific to iOS 11?

If the guards can’t be avoided we should document each one.  Otherwise it’ll be hard to understand why they’re necessary.

Reply:
Ps: ideally we should try to avoid branching the code by iOS version.

Reply:
This code only work for iOS11 because they changed the navigation bar to use stack view internally to distribute the items. on iOS 10 and before we still need the old code.